README.md

```diff
@@ -109,4 +109,4 @@ This project is developed and maintained by the repo owner and volunteers from [
 
 ## License
 
-Distributed under the MIT License. See `LICENSE` for more information.
\ No newline at end of file
+Distributed under the CeCILL and GPLv3 Licenses. See `LICENSE` for more information.
```

Comment:
CeCILL and GPLv3 Licenses --> CeCILL and AGPLv3 Licenses

Reply:
Thanks for catching these :)